desktop/app/mainWindow/index.js

```diff
@@ -59,8 +61,15 @@ function showAppWindow() {
 	mainWindow.webContents.loadURL( `file://${ getPath( 'index.html' ) }` );
 
 	mainWindow.setBrowserView( mainView );
-	mainView.setBounds( { ...bounds, ...{ x: 0, y: 38 } } );
-	mainView.setAutoResize( { horizontal: true, vertical: true } );
+	mainView.setBounds( { ...bounds, ...{ x: 0, y: TITLE_BAR_HEIGHT } } );
```

Comment:
The one thing I noticed is that it seemed to cut off some elements if they were stuck to the bottom of the window.

I think something like this would solve it.
```
mainView.setBounds( {
		...bounds,
		...{
			x: 0,
			y: TITLE_BAR_HEIGHT,
			width: bounds.width,
			height: bounds.height - TITLE_BAR_HEIGHT,
		},
	} );
```

Reply:
Thanks for catching that! Fixed in ece759427cdfb9fb73a36a7c5859f56a426fd7e5